starter_code/components.js

```diff
@@ -0,0 +1,33 @@
+class Image extends React.Component {
+  constructor(){
+    super();
+    this.state = {show: false};
+  }
```

Comment:
Don't forget to accept `props` as an argument to the constructor function and then pass it to `super`! Like so:

```js
constructor(props) {
  super(props);
  this.state = { show: false };
}
```